packages/account/src/Sections/Profile/PhoneVerification/didnt-get-the-code-modal.tsx

```diff
@@ -0,0 +1,122 @@
+import React from 'react';
+import { Button, Modal, Text } from '@deriv-com/quill-ui';
+import { Localize } from '@deriv/translations';
+import { observer, useStore } from '@deriv/stores';
+import { VERIFICATION_SERVICES } from '@deriv/shared';
+import { useGetPhoneNumberOTP } from '@deriv/hooks';
+
+type TDidntGetTheCodeModal = {
+    should_show_didnt_get_the_code_modal: boolean;
+    setShouldShowDidntGetTheCodeModal: (value: boolean) => void;
+    setStartTimer: (value: boolean) => void;
+    phone_verification_type: string;
+    setOtpVerification: (value: { show: boolean; phone_verification_type: string }) => void;
+};
+
+const DidntGetTheCodeModal = observer(
+    ({
+        should_show_didnt_get_the_code_modal,
+        setShouldShowDidntGetTheCodeModal,
+        setStartTimer,
+        phone_verification_type,
+        setOtpVerification,
+    }: TDidntGetTheCodeModal) => {
+        const { requestOnSMS, requestOnWhatsApp, ...rest } = useGetPhoneNumberOTP();
+        const { ui } = useStore();
+        const { is_mobile } = ui;
```

Comment:
```suggestion
        const { is_mobile } = ui;
```

useDevice?

Reply:
We will do it in parent PR when this branch is merged 🙏